src/Components/Navbar/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './style.css';
+
+export default function Navbar() {
+  return (
+    <>
```

Comment:
you don't need Fragment here. your component is already wrapped in <div className='navbar'>
also true for other components